src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/DashboardViewModel.cs

```diff
@@ -87,7 +87,7 @@ private void OnReplacementsComputed(object sender, IInlineRenameReplacementInfo
             {
                 _errorText = string.IsNullOrEmpty(session.ReplacementText)
                     ? null
-                    : string.Format(EditorFeaturesResources.IsNotAValidIdentifier, GetTruncatedName(session.ReplacementText));
+                    : string.Format(EditorFeaturesResources.TheNewNameIsNotAValidIdentifier);
```

Comment:
Drop the `string.Format()` call since the resource doesn't take parameters anymore.